OpenTabletDriver.Web/Views/Wiki/FAQ/Linux.cshtml

```diff
@@ -2,67 +2,122 @@
     ViewBag.Title = "Linux FAQ";
 }
 
-<h3 class="wiki-nav-item pb-2" id="ghost-cursor">
-    Teleporting cursor position
+<h3 class="wiki-nav-item pb-2" id="tablet-not-detected">
+    My tablet is not detected?
 </h3>
 <div class="ms-3">
     <p>
-        This occurs because another program is reading the tablet at the same time as OpenTabletDriver.
-        <small>
-            <a class="link-light" href="https://github.com/OpenTabletDriver/OpenTabletDriver/issues/68">#68</a>
+        This can be caused by lots of things on Linux, for sanity reasons make sure that your tablet is actually plugged in to your computer,
+        you can usually check this with a gui utility or with an application like <code>lsusb</code>,
+        once you are 100% sure your tablet is connected to your computer, ensure that your specific tablet make and model is supported by checking our list of supported tablets
+        <a href="/Tablets">here</a>.<br/><br/>
+
+        We currently do not support tablets that connect directly to a computer via bluetooth, and have limited support for ones that use wireless USB receivers,
+        so please make sure your tablet is connected via cable.<br/>
+        <small class="text-muted">
+            Note: If you installed OpenTabletDriver with a package manager, you usually just need to reboot,
+            as the driver may not work out of the box on some configurations.
```

Comment:
should only need replug at most on new OS'es, I believe @X9VoiD fixed the (hopefully) final issues with needing a reboot which was with Fedora, but correct me if I'm wrong. I don't mind suggesting reboot as a last option but it should not be our first recommendation - Linux kinda prides itself with never needing a reboot so I think its worth trying to push for that as much as is fair.